freud/cluster.pyx

```diff
@@ -159,14 +157,16 @@ cdef class ClusterProperties(_Compute):
     Given a set of points and cluster ids (from :class:`~.Cluster` or another
     source), this class determines the following properties for each cluster:
 
-     - Center of mass
+     - Geometric centers
+     - Centers of mass
      - Gyration tensor
+     - Moment of inertia tensor
      - Size (number of points)
+     - Mass
```

Comment:
Does mass refer to the total mass of each cluster, or average mass of a particle in each cluster? Please clarify in the docs.

Reply:
Mass of each cluster. I will clarify in the docs.